src/console_cmds.cpp

```diff
@@ -1154,6 +1172,35 @@ DEF_CONSOLE_CMD(ConExec)
 	return true;
 }
 
+DEF_CONSOLE_CMD(ConSchedule)
+{
+	if (argc < 3 || std::string_view(argv[2]) != "on-next-calendar-month") {
+		IConsolePrint(CC_HELP, "Schedule a local script to execute later. Usage: 'schedule <script> on-next-calendar-month'.");
+		return true;
+	}
+
+	/* Check if the file exists. It might still go away later, but helpful to show an error now */
```

Comment:
```suggestion
	/* Check if the file exists. It might still go away later, but helpful to show an error now. */
```

VERY small nitpick. Hold true for all other comments too. Sentences end with a dot. :D

Reply:
Have added the dots

(There is one comment that is maybe a title? "Scheduled execution handling"... it does have a dot but maybe it shouldn't since it's not really a sentence?)

Reply:
In general we just say: always add a dot, to remove the discussion whether it is a title or not. But it is all good. If we get 90% dots, I succeeded in my job :D